appengine/standard_python37/bigquery/templates/query_result.html

```diff
@@ -1,21 +1,31 @@
-<!doctype <!DOCTYPE html>
-<html>
-<head>
-    <meta charset="utf-8" />
-    <title>Query Result</title>
-</head>
-<body>
-    <table>
-        <tr>
-            <th>URL</th>
-            <th>View Count</th>
-        </tr>
-        {% for result in results %}
-        <tr>
-            <td>{{ result[0] }}</td>
-            <td>{{ result[1] }}</td>
-        </tr>
-        {% endfor %}
-    </table>
-</body>
-</html>
\ No newline at end of file
+<!DOCTYPE html>
+{#
```

Comment:
Is it intended to make this text not appear in the page? If so, the syntax for HTML comments is different:

<!-- this is a comment -->